packages/headless/src/features/product-listing/product-listing-analytics.actions.ts

```diff
@@ -0,0 +1,32 @@
+import {Result} from '../..';
+import {ProductListingAnalyticsProvider} from '../../api/analytics/product-listing-analytics';
+import {
+  AnalyticsType,
+  ClickAction,
+  documentIdentifier,
+  makeAnalyticsAction,
+  partialDocumentInformation,
+  ProductListingAction,
+  validateResultPayload,
+} from '../analytics/analytics-utils';
+
+export const logProductListing = (): ProductListingAction =>
+  makeAnalyticsAction(
+    'analytics/productListing/load',
+    AnalyticsType.Search,
+    (client) => client.makeInterfaceLoad(),
+    (getState) => new ProductListingAnalyticsProvider(getState)
+  );
+
+export const logProductListingOpen = (result: Result): ClickAction =>
```

Comment:
Should we call this `logDocumentOpen` for consistency? The `ProductListingOpen` naming suggests the PLP was opened, but this event is more for a click on a specific product,

Reply:
+1